server/social.coffee

```diff
@@ -400,15 +400,17 @@ module.exports = exports = (log, loga, argv) ->
         false
 
       app.all '*', (req, res, next) ->
-        # todo: think about assets??
+        # don't protect site flag,
+        return next() if req.url is '/favicon.png'
         return next() unless /\.(json|html)$/.test req.url
 
         # prepare to examine remote server's forwarded session
         res.header 'Access-Control-Allow-Origin', req.get('Origin')||'*'
         res.header 'Access-Control-Allow-Credentials', 'true'
-        return next() if isAuthorized(req) || allowedToView(req)
+        # protect unclaimed by adding "add owner isnt ''" - maybe via parameter
+        return next() if isAuthorized(req) or allowedToView(req)
         return res.redirect("/view/#{m[1]}") if m = req.url.match /\/(.*)\.html/
-        return res.json([]) if req.url == '/system/sitemap.json'
+        return res.json(['Login Required']) if req.url == '/system/sitemap.json'
```

Comment:
So this will return with status 200, is that important where we normally fetch sitemaps? I think where scripts fetch sitemaps it will probably throw some javascript error. Then so would a 403 be ok or would we need more work elsewhere to handle them?

Reply:
Using `res.status(403).json([])` was my first thought. But, to handle nicely in the siteAdapter would likely require a rewrite to use `fetch()`. Something that would take some time. Just setting the status is not an option, as that puts the client into a state of being unable to fetch the sitemap that it doesn't recover from.